vm/src/builtins/int.rs

```diff
@@ -751,6 +755,78 @@ impl Hashable for PyInt {
     }
 }
 
+impl AsNumber for PyInt {
+    const AS_NUMBER: PyNumberMethods = PyNumberMethods {
+        add: Some(|number, other, vm| Self::np_int_op(number, other, |a, b| a + b, vm)),
+        subtract: Some(|number, other, vm| Self::np_int_op(number, other, |a, b| a - b, vm)),
+        multiply: Some(|number, other, vm| Self::np_int_op(number, other, |a, b| a * b, vm)),
+        remainder: Some(|number, other, vm| Self::np_general_op(number, other, inner_mod, vm)),
+        divmod: Some(|number, other, vm| Self::np_general_op(number, other, inner_divmod, vm)),
+        power: Some(|number, other, vm| Self::np_general_op(number, other, inner_pow, vm)),
+        negative: Some(|number, vm| {
+            Self::number_downcast(number)
+                .value
+                .clone()
```

Comment:
https://docs.rs/num/latest/num/struct.BigInt.html#impl-Neg-1

Neg is also implemented for `&BigInt`. This clone is redundant.